Chary/Assignments/Functions/2_reverse_the_string.py

```diff
@@ -0,0 +1,14 @@
+#Write a program reverse a given input string
+
+def reverse_string(x):
+    y=""
+    z=len(x)
+    for i in range(-1,-(z+1),-1):
```

Comment:
Start using directly  the syntax range(-1, -(len(x)-1), -1) avoid additional variable unless its not used further

Reply:
fixed